src/clients/mod.rs

```diff
@@ -4,6 +4,7 @@ use crate::*;
 use ::momento::MomentoError;
 use async_channel::Receiver;
 use std::io::{Error, ErrorKind, Result};
+use std::time::Instant;
```

Comment:
Just so I understand, this is here because clients/* import super::*? Because I don't see any local changes which would require it...

Reply:
Yes, just felt like a convenient place to pull in the instant type we want to use.